### ISSUE TITLE
" Lizard TP 10M " Problems with the correct use of the entire loading area. 

### ISSUE THREAD

**Issue by SSlz** [ddewar/FS22_UniversalAutoloadModhubAddon#85]
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=249931&title=fs2022

With this low-loader, only the front area above the steering axle is used when loading with pallets. The rest of the loading area is unused. Is there a way to use the entire loading area?

Vielen Dank

**Comment by SSlz** [comment]
Yes, it works. But now I have the problem that in some situations the steering axle loses traction. Is there a way to distribute the load better?

Unloading and reloading does not bring the desired success.

**Comment by ddewar** [comment]
> https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=249931&title=fs2022
> 
> With this low-loader, only the front area above the steering axle is used when loading with pallets. The rest of the loading area is unused. Is there a way to use the entire loading area?
> 
> Vielen Dank

Hey there, do you have a screenshot of what you're asking about? If you are only loading a few pallets, it will load them from the front. If you want to distribute more when you only have a few pallets try using the horizontal loading mode to make just one layer of pallets.
